src/de/jost_net/JVerein/gui/view/MailVorlageDetailView.java

```diff
@@ -40,9 +45,16 @@ public void bind() throws Exception
     SimpleContainer t = new SimpleContainer(getParent(), true);
     t.addPart(control.getTxt());
 
+    LesefeldAuswerter lesefeldAuswerter = new LesefeldAuswerter();
+    lesefeldAuswerter.setLesefelderDefinitionsFromDatabase();
+    lesefeldAuswerter.setMap(new MitgliedMap().getMap(new MitgliedDummy(), null));
+    lesefeldAuswerter.evalAlleLesefelder();
```

Comment:
Verwende hier:
`Mitglied m = Einstellungen.getDBService().createObject(MitgliedImpl.class, null);`
Dann ist kein Dummy Mitglied nötig
in z56 dann
`new OpenInsertVariableDialgoAction(),new MitgliedMap().getMap(m, null), false, "bookmark.png");`

Reply:
Ich habe stattdessen in MitgliedMap ein MitgliedDummy verwendet. Ich persönlich finde es intuitiver ein Mock-Objekt in solchen Fällen zu verwenden, als die Beispiel-Werte In-Place festzulegen. So kann jetzt auch, sollten einmal Beispiel-Werte außerhalb von MitgliedMap gebraucht werden einfach ein Dummy erzeugt werden.

Reply:
Ansonsten kann da gerne auch noch @dippeal oder @JohannMaierhofer ne Meinung zu abgeben. Wenn die Mehrheit das so lassen möchte wie es ist, ändere ich es gerne auch wieder zurück.